### PR TITLE
🐛 Source Pinterest: fix `boards` schema with correct format for `date-time`

### PR DESCRIPTION
**Pull request by bazarnov**
## What
Resolving: https://github.com/airbytehq/oncall/issues/1848

## How
- edited the `boards` schema with `date-time` format instead of `datetime`

## 🚨 User Impact 🚨
No impact is expected. This is not breaking change.

## Pre-merge Checklist
*Expand the relevant checklist and delete the others.*

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Connector version has been incremented
    - [ ] Version has been bumped according to our [Semantic Versioning for Connectors](https://docs.airbyte.com/contributing-to-airbyte/#semantic-versioning-for-connectors) guidelines
    - [ ] `Dockerfile` has updated version
- [ ] Documentation updated
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] Changelog updated in `docs/integrations/<source or destination>/<name>.md` with an entry for the new version. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)

- [ ] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing
- [ ] New Connector version released on Dockerhub and connector version bumped by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)

</details>

### CODE REVIEW

**Review by artem1205 — APPROVED** [review]

File: airbyte-integrations/connectors/source-pinterest/acceptance-test-config.yml [hunk 1]
```diff
@@ -41,10 +41,10 @@ acceptance_tests:
   discovery:
     tests:
       - backward_compatibility_tests_config:
-          disable_for_version: 0.2.3
+          disable_for_version: 0.2.5
         config_path: secrets/config.json
       - backward_compatibility_tests_config:
-          disable_for_version: 0.2.3
+          disable_for_version: 0.2.5
```
Comment (erohmensing) [inline airbyte-integrations/connectors/source-pinterest/acceptance-test-config.yml:47]: Why do we need to disable the backwards compatibility tests if its not a breaking change?

Comment (bazarnov) [reply airbyte-integrations/connectors/source-pinterest/acceptance-test-config.yml:47]: Correct me if, I'm wrong, but changing the `format` in the schema leads to a crash of CAT but this is not a braking change for the Customer. No?

Comment (bazarnov) [reply airbyte-integrations/connectors/source-pinterest/acceptance-test-config.yml]: @erohmensing It's actually not needed. I've removed the `backward_compatibility_tests_config` from `acceptance-test-config.yaml`, the changes are not breaking.

File: airbyte-integrations/connectors/source-pinterest/metadata.yaml [hunk 2]
```diff
@@ -10,7 +10,7 @@ data:
   connectorSubtype: api
   connectorType: source
   definitionId: 5cb7e5fe-38c2-11ec-8d3d-0242ac130003
-  dockerImageTag: 0.2.4
```
Comment (erohmensing) [inline airbyte-integrations/connectors/source-pinterest/metadata.yaml:13]: Note for posterity if anyone is confused why this skipped a version - these metadata files were generated once a little while ago. They're currently not in use anywhere, so they may go stale, and that's okay. The place to update for the moment is still the [dockerfile](https://github.com/airbytehq/airbyte/pull/25548/files#diff-6eeb808f0899bd5167d03b761160d0ec6566f692829dc5317a5e935bcfdb7151R37) as has been done in this PR. 

Eventually we'll switch over to publish on merge, and that publish workflow will bump the dockerfile and the metadata version in sync. that's when the metadata files will start always being in sync (and we'll re run generation again to make sure that nothing is out of date before starting this process). 

Cc @bnchrch to check my work

**Review by erohmensing — APPROVED** [review]

File: airbyte-integrations/connectors/source-pinterest/acceptance-test-config.yml (hunk 1 above)
Comment (erohmensing) [reply airbyte-integrations/connectors/source-pinterest/acceptance-test-config.yml]: Thanks, makes me feel better!

**Review by lazebnyi — APPROVED** [review]
LGTM!

**Pull request merged**